src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
```

Comment:
The constructor of the `Ball` class should have an access modifier to define its visibility properly. In Java, it's a good practice to explicitly define the access level rather than relying on the default package-private access.